doc/faq/virtualenv_faq.rst

```diff
@@ -21,17 +21,18 @@ installed.
 
 Otherwise, the situation (at the time of writing) is as follows:
 
-============= ========================== =================================
-GUI framework pip-installable?           conda or conda-forge-installable?
-============= ========================== =================================
-PyQt5         yes                        yes
-------------- -------------------------- ---------------------------------
-PyQt4         PySide: on Windows and OSX yes
-------------- -------------------------- ---------------------------------
-PyGObject     yes [#]_                   on Linux
-------------- -------------------------- ---------------------------------
-wxPython      yes [#]_                   yes
-============= ========================== =================================
+========= ========= ================ =================================
+framework bindings  pip-installable? conda or conda-forge-installable?
+========= ========= ================ =================================
+Qt5       PyQt5     yes              yes
+Qt5       PySide2   yes [#]_         yes
+Qt4       PyQt4     no               yes
+Qt4       PySide    Windows          yes
```

Comment:
Should this be Windows and OSX?

Reply:
For whatever reason there are no OSX wheels at https://pypi.org/project/PySide/#files, but they are there for the previous release (1.2.2) (https://pypi.org/project/PySide/1.2.2/#files), which I guess is good enough.  Restored the mention of OSX.